demisto_sdk/commands/pre_commit/hooks/ruff.py

```diff
@@ -43,12 +38,12 @@ def prepare_hook(
                 f"--target-version={self._python_version_to_ruff(python_version)}"
             )
             safe_update_hook_args(hook, target_version)
-            if github_actions:
+            if os.getenv("GITHUB_ACTIONS", False):
```

Comment:
cant be fixed with modes? Dont think this is needed

Reply:
i don't think it's worth another mode in this case, we can detect it by env variable.